src/components/MDX/Sandpack/SandpackRoot.tsx

```diff
@@ -70,17 +71,19 @@ function SandpackRoot(props: SandpackProps) {
   const codeSnippets = Children.toArray(children) as React.ReactElement[];
   const files = createFileMap(codeSnippets);
 
-  files['/styles.css'] = {
-    code: [sandboxStyle, files['/styles.css']?.code ?? ''].join('\n\n'),
-    hidden: !files['/styles.css']?.visible,
+  files['/src/styles.css'] = {
+    code: [sandboxStyle, files['/src/styles.css']?.code ?? ''].join('\n\n'),
+    hidden: !files['/src/styles.css']?.visible,
   };
 
   return (
     <div className="sandpack sandpack--playground w-full my-8" dir="ltr">
       <SandpackProvider
-        template="react"
-        files={files}
+        files={{...template, ...files}}
         theme={CustomTheme}
+        customSetup={{
```

Comment:
Does this change some of the config for how Sandpack is rendered? It looks like the "download button" is now missing after this PR.

Link to missing button in PR preview: https://react-dev-git-fork-danilowoz-draft-flamboyant-rain-fbopensource.vercel.app/learn/tutorial-tic-tac-toe#what-are-you-building

Link to codesandbox with download button in current deployed react.dev: https://react.dev/learn/tutorial-tic-tac-toe#what-are-you-building

Screenshot of codesandbox with download button in current deployed react.dev: 
<img width="259" alt="Screenshot 2023-12-12 at 1 50 34 PM" src="https://github.com/reactjs/react.dev/assets/7158882/3d192cfd-abf5-414c-b2ae-e4da5d1615c5">

Reply:
Good catch! The download button is conditionally rendered based on the App.js path, which I just changed.